usr/share/rear/output/USB/Linux-i386/300_create_grub.sh

```diff
@@ -51,7 +51,27 @@ if [ ! -d "$usb_boot_dir" ] ; then
     mkdir -p $v "$usb_boot_dir" || Error "Failed to create USB boot dir '$usb_boot_dir'"
 fi
 DebugPrint "Installing GRUB2 as USB bootloader on $RAW_USB_DEVICE"
-$grub_install_binary --boot-directory=$usb_boot_dir --recheck $RAW_USB_DEVICE || Error "Failed to install GRUB2 on $RAW_USB_DEVICE"
+# Set default USB_GRUB2_INSTALL_OPTIONS only if there are no USB_GRUB2_INSTALL_OPTIONS set:
+test "$USB_GRUB2_INSTALL_OPTIONS" || USB_GRUB2_INSTALL_OPTIONS=""
+# grub-install defaults to '--target=x86_64-efi' when the system is booted with EFI.
+# So it would fail to install a legacy BIOS (or 32bit) GRUB2 without setting the --target parameter.
+# ("man grub2-install" tells "TARGET platform [default=i386-pc]" but this is more like a fallback.)
+# So setting explicitly a legacy BIOS target is needed when the system is booted with EFI,
+# see https://github.com/rear/rear/issues/2883
+if is_true $USING_UEFI_BOOTLOADER ; then
+    # TODO: only call grub-install if legacy boot install is explicitly requested
+    # TODO: use a switch case based on the target (uname -m) and possibly other info?
+    # Enforce legacy BIOS installation since EFI was handled in 100_create_efiboot.sh
+    # see https://github.com/rear/rear/issues/2883
+    # Set a GRUB2 target only if there is no '--target=' in USB_GRUB2_INSTALL_OPTIONS
+    # (according to "man grub2-install" of grub2-2.06 on openSUSE Leap 15.4
+       '--target=TARGET' is the only possible syntax to specify a GRUB2 target)
```

Comment:
looks like this line is missing a #

Reply:
Ouch!

@DEvil0000 
thank you for spotting it!

I cannot explain why I did not see it myself :-(

Fixed via
https://github.com/rear/rear/pull/2905/commits/74eecae4bdc27f23276e639c64ab5f6d2e446760